lib/Runtime/Library/DataView.h

```diff
@@ -55,6 +55,34 @@ namespace Js
             return static_cast<DataView*>(aValue);
         }
 
+        static uint32 ToIndex (Var value, int32 errorCode, ScriptContext *scriptContext)
```

Comment:
Especially since this is currently specific to DataView, shouldn't this method not take an error code and always throw InvalidOffset?